_data/team/roles/documentation.yml

```diff
@@ -0,0 +1,18 @@
+role: Documentation
+tasks: 
+  - Managing UserGuide repositories and main core documentation
+  - Managing MDAnalysis Sphinx theme
+  - Managing documentation-related utilities and architecture
+  - Managing new issues and PRs related to documentation
+  - Keeping documentation updated
+  - Responding to documentation issues in sub-projects
+current_leads:
+  - Lily Wang
+  - Rocco Meli
+current_members:
+  - Fiona Naughton
```

Comment:
Only picking one example here - this structure doesn't make sense to me.

You have Fiona specificly outlined and then "All core devs" - this reads to me as "Fiona is not a core dev", which I don't think is the intent here.

I think you either have to choose to have specific named folks or leave it as "all core devs".

Reply:
@fiona-naughton I'll remove you from members for now, please sign yourself up if you would like to lead docs!